spec/libsass-closed-issues/issue_83/error-dart-sass

```diff
@@ -1,5 +1,10 @@
 Error: Missing argument $color.
-  @include colors {
-  ^^^^^^^^^^^^^^^^^^^
-  spec/libsass-issues/issue_83/input.scss 7:3  colors()
-  spec/libsass-issues/issue_83/input.scss 7:3  root stylesheet
+  ,
+7 | /   @include colors {
+8 | |     color: $color;
+9 | |   }
+10| |   border-width: 10px;
+  | '--^
```

Comment:
It looks like the error indicator "^" moved to a more confusing location. Should it be before the line 
`border-width: 10px;`

Reply:
Great! SG.